skimage/io/tests/test_tifffile.py

```diff
@@ -57,7 +57,7 @@ def roundtrip(self, dtype, x, use_pathlib=False):
 
         if use_pathlib:
             fname = pathlib.Path(fname)
-        imsave(fname, x, check_contrast=False)
+        imsave(fname, x, check_contrast=False, photometric = 'minisblack')
```

Comment:
Actually I we don't have to worry about this that much. The way I read the deprecation warning is that tifffile will just change the internal storage layout. For round tripping this should not have much of an impact and if it has it will be detected by this test.

I have a slight tendency to keep this as is and not make this explicit by adding the `photometric` parameter. I think there is value in testing the round tripping for the case using the default arguments. What do you think @rosdyana?

Fyi, we are thinking about deprecating the `skimage.io` module all together.

Reply:
Sure @lagru , I think we can close this PR and keep using the default arguments. There is no benefit in fixing the warning since it will be deprecated.

Reply:
this PR https://github.com/scikit-image/scikit-image/pull/6571 was similar